.github/actions/check-files/action.yml

```diff
@@ -33,6 +33,13 @@ runs:
           fi
         done
 
+        for file in $files; do
```

Comment:
for now this is fine, I want to eventually replace this entire action with https://github.com/dorny/paths-filter